public/javascripts/payment_inside_client.js

```diff
@@ -101,12 +101,12 @@ function loadPaymentPages(data, prepopulateFields, req) {
   params['param_supportedTypes'] =
     'AmericanExpress,JCB,Visa,MasterCard,Discover';
   params['url'] = data.url;
-  let gtOptions = req.gatewayOptions.split(";");
+  let gtOptions = req.gatewayOptions.split(“;”);
   gtOptions.forEach((gtOption) => {
-    let ref = gtOption.split(":");
-    let pass = "param_gwOptions_" + ref[0];
+    let ref = gtOption.split(“:”);
```

Comment:
Can you check if the quotes is correct?  after finish the code change, Please test it , make it works